### ISSUE TITLE
Fading

### ISSUE THREAD

**Issue by brs17** [pop-os/gnome-shell-theme#15]
```
 * Distribution - (run ```lsb_release -a```)
Pop!_OS 18.04
 * Gtk+ 3.0 version - (run ```apt policy libgtk-3-0```)
3.22.29-3ubuntu1
 * Desktop environment - 
Gnome
 * Pop version - (run ```head -n 3 /usr/share/gnome-shell/theme/pop.css```)
4.0.0
 ```
**Actual issue**
When more than 6 different applications are open, in the alt-tab display, the last program is half faded out.

![bug](https://user-images.githubusercontent.com/7460774/38900559-07bda5ba-4259-11e8-87ed-37286bff2ff2.png)

**Steps to reproduce**
Open 7 different applications.
press <kbd>alt</kbd>+<kbd>tab</kbd>
See bug.

**Expected behavior**
Last window doesn't fade.

**Behavior in Pop!_OS 17.10**
Does not fade out last program.

**Other Note (feature-request, question, etc...)**
This bug has been recreated on a bonw11, gaze12, and oryp3. 
Changing screen resolution does not change the number of programs required to recreate this bug.

**Comment by brs17** [comment]
In a live image, I just found this issue to happen on just two different programs open.

**Comment by isantop** [comment]
Fixed in #19
